provision/acc_provision/acc_provision.py

```diff
@@ -353,17 +384,67 @@ def config_adjust(args, config, prov_apic, no_random):
                 node_svc_subnet,
             ],
         },
+        "cf_config": {
+            "default_endpoint_group": {
+                "tenant": tenant,
+                "app_profile": "cloudfoundry",
+                "group": "cf-app-default",
+            },
+            "node_subnet_cidr": "%s/%s" % cidr_split(node_subnet)[3:],
+            "node_epg": "cf-node",
+            "app_ip_pool": [
+                {
+                    "start": cidr_split(pod_subnet)[0],
+                    "end": cidr_split(pod_subnet)[1],
+                }
+            ],
+            "app_subnet": "%s/%s" % cidr_split(pod_subnet)[2::2],
+            "dynamic_ext_ip_pool": [
+                {
+                    "start": cidr_split(extern_dynamic)[0],
+                    "end": cidr_split(extern_dynamic)[1],
+                },
+            ],
+            "static_ext_ip_pool": [
+                {
+                    "start": cidr_split(extern_static)[0],
+                    "end": cidr_split(extern_static)[1],
+                },
+            ],
+            "node_service_ip_pool": [
+                {
+                    "start": cidr_split(node_svc_subnet)[0],
+                    "end": cidr_split(node_svc_subnet)[1],
+                },
+            ],
+            "node_service_gw_subnets": [
+                node_svc_subnet,
+            ],
+            "api_port": 9900,
+        },
         "registry": {
             "configuration_version": token,
         }
     }
+    adj_config["cf_config"]["node_network"] = (
+        "%s|%s|%s" % (
+            tenant,
+            adj_config['cf_config']['default_endpoint_group']['app_profile'],
+            adj_config['cf_config']['node_epg']))
+
     return adj_config
 
 
-def config_validate(config):
+def config_validate(flavor, config):
     def Raise(exception):
         raise exception
     required = lambda x: True if x else Raise(Exception("Missing option"))
+    required_if_kube = (lambda x:
```

Comment:
ideally these checks would not be expressed hard-coded based on template type; this is identical to matching on flavor